packages/teleport/src/Kubes/Kubes.tsx

```diff
@@ -32,21 +32,29 @@ export default function Container() {
   return <Kubes {...state} />;
 }
 
+const docUrl =
+  'https://goteleport.com/docs/kubernetes-access/guides/multiple-clusters/';
+
 export function Kubes(props: State) {
-  const { kubes, attempt } = props;
+  const { kubes, attempt, user, showButton } = props;
 
   return (
     <FeatureBox>
       <FeatureHeader alignItems="center" justifyContent="space-between">
         <FeatureHeaderTitle>Kubernetes</FeatureHeaderTitle>
+        {showButton && (
+          <ButtonPrimary as="a" width="240px" target="_blank" href={docUrl}>
```

Comment:
one more change, for links, if we dont use our `Link` component, we need to manually set `rel="noreferrer"`: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noreferrer

Reply:
oh, okay to hide the referrer information.